baybe/surrogates/base.py

```diff
@@ -48,7 +47,7 @@
 """
 
 
-@define(slots=False)
```

Comment:
Not that I'm aware of ... I disabled the slotting during one of the last refactorings only due to the way the `autoscale` and `catch_constant_target` decorators were redesigned, but already knowing that this was only a temporary solution. The deactivation was necessary because the added a new attribute to the class instances, which is not possible with slots. And here you already have a partial answer why I think the new version is nicer / we should at least change that part in the decorators. 

Confident that it won't reintroduce the problems? Are you kidding 😄? You know very well how random our serialization stuff and other errors are in our current pipeline 🙈  But I'd say I currently see no reason against reactivating

Reply:
then we need a way to remember to quickly thes this with simulation and nonGP surrogates once everything is ready